### PR TITLE
Correctly import default get from lodash.get

### PR DESCRIPTION
**Pull request by joshlevinson**
`lodash.get` exports the get function as its default export. Importing it as though it were an individually exported module (rather than the default) doesn't work–it results in an undefined function error.

### CODE REVIEW

**Comment by joshlevinson** [comment]
For the sake of getting builds to pass, merged #67 into this branch.

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "is-node-fn": "^1.0.0",
     "lodash.get": "^4.4.2",
     "lodash.merge": "^4.6.0",
-    "normalizr": "^3.2.3",
+    "normalizr": "^2.3.1",
```
Comment (sdgluck) [inline package.json]: What's the reason for downgrading `normalizr`?

Comment (sdgluck) [reply package.json]: My bad- this is from #67. Ignore me.

**Comment by sdgluck** [comment]
Thank you! 😸

**Pull request merged**